recipes/scalecodec/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
You don't need to include an extra copy of the license here. LICENSE is included in the source tar.